dags/common/utils.py

```diff
@@ -291,6 +290,58 @@ def article_type_filter(article_type: str, types: list) -> bool:
         return False
     return article_type.lower() in set(t.lower() for t in types)
 
+def title_filter(case_report: dict) -> bool:
+    """Checks to see if title shows a cardiology case_report
+    Args:
+        case_report (dict): json file
+        terms (list): list of cardiology terms
+    Returns:
+        bool: returns whether title is cardiology-related
+    """
+    title = case_report.get("title")
+    terms = ["heart", "cardiology", "heartrhythm", "cardiovascular", "heart rhythm", "cardio"]
+    temp = []
+    if isinstance(title, str):
+        temp.extend([x.upper() for x in terms if x.lower() in title.lower()])
+    if len(temp) > 0:
+        return True
+    else:
+        return False
+
+def text_filter(case_report: dict) -> bool:
```

Comment:
looks like text_filter and title_filter essentially do the same thing. I'd rename the existing title_filter to text_filter and use it for both the title and text fields.